<launch>
	<node name="motor_controller" pkg="whatever" output="screen" type="motor_controller" />
	<node name="node_controller" pkg="whatever" output="screen" type="node_controller" />
	<node name="remote_monitor" pkg="whatever" output="screen" type="remote_monitor" />
	<node name="debug_makara" pkg="whatever" output="screen" type="debug_makara" />
	<node name="pid_controller" pkg="whatever" output="screen" type="pid_controller" />
</launch>
